test/RGGuardValidator.test.ts

```diff
@@ -32,6 +32,29 @@ describe("GuardValidator", () => {
         }).toThrow("You must provide at least one rule to the RGGuardValidator");
     });
 
+    it("should accept custom rules", () => {
+        const customRules = [
+            RULES[ComplianceFramework.HIPAA_SECURITY],
+            RULES[ComplianceFramework.PCI_DSS_3_2_1],
+        ];
+        const guard = new RGGuardValidator({ rules: customRules });
+        expect(guard.rules).toEqual(customRules);
+    });
+
+    it("should set throwErrorOnComplianceFail to true by default", () => {
+        const guard = new RGGuardValidator();
+        expect(guard).toBeDefined();
+        expect(guard.name).toBeDefined();
+    });
```

Comment:
This is not doing anything to check the throwErrorOnComplianceFail has been set. Need to check the property has been set or run the validation to check for the output